tools/reproduce_comparison/Jenkinsfile

```diff
@@ -59,7 +59,7 @@ pipeline {
                     jsonJobParams.BUILD_CONFIGURATION.SCM_REF = scmRef
                     jsonJobParams.BUILD_CONFIGURATION.BUILD_REF = buildRef
                     jsonJobParams.BUILD_CONFIGURATION.BUILD_ARGS += " --build-reproducible-date ${buildTimeStamp}"
-                    jsonJobParams.BUILD_CONFIGURATION.ENABLE_REPRODUCIBLE_COMPARE = "false"
+                    jsonJobParams.BUILD_CONFIGURATION.ENABLE_REPRODUCIBLE_COMPARE = false
```

Comment:
think the reason with "" is to treat it as String, since textParam only support String.  Can test and see if it work here

Reply:
ENABLE_REPRODUCIBLE_COMPARE is defined as Boolean https://github.com/adoptium/jenkins-helper/blob/master/src/common/IndividualBuildConfig.groovy#L55